pkg/insights/metrics.go

```diff
@@ -35,29 +35,30 @@ func init() {
 	MustRegisterMetrics(RecommendationCollector)
 }
 
-type InsightsRecommendationCollector struct {
+// Collector collects insights recommendations
+type Collector struct {
```

Comment:
I like the original name more to be honest. It implements the Prometheus collector pattern.

Reply:
This is how the Go style/code convention works: https://google.github.io/styleguide/go/best-practices.html

Maybe we could change it to `RecommendationCollector` instead?

Reply:
Well we don't follow the Go conventions in another places anyway... :) Anyway the alert is called `InsightsRecommendationActive` so the naming here seems to me quite reflecting. ...

Reply:
Are you sure? I don't get any other alert from the code analysis tool. Can you point out an example?

Reply:
Actually, we can't rename it to `RecommendationCollector` 'cause we already have an exported variable with this name. I would keep Golang code-style recommendation and leave the name as `Collection`.